packages/fast-tooling-react/src/css-editor/editor.schema.json

```diff
@@ -5,67 +5,10 @@
     "id": "@microsoft/fast-tooling-react/css-editor",
     "type": "object",
     "properties": {
-        "data": {
-            "title": "CSS Values",
-            "type": "object",
-            "properties": {
-                "position": {
-                    "title": "Position",
-                    "type": "string",
-                    "enum": [
-                        "absolute",
-                        "static"
-                    ]
-                },
-                "left": {
-                    "title": "Left",
-                    "type": "string"
-                },
-                "right": {
-                    "title": "Right",
-                    "type": "string"
-                },
-                "top": {
-                    "title": "Top",
-                    "type": "string"
-                },
-                "bottom": {
-                    "title": "Bottom",
-                    "type": "string"
-                },
-                "marginTop": {
-                    "title": "Margin top",
-                    "type": "string"
-                },
-                "marginBottom": {
-                    "title": "Margin bottom",
-                    "type": "string"
-                },
-                "marginLeft": {
-                    "title": "Margin left",
-                    "type": "string"
-                },
-                "marginRight": {
-                    "title": "Margin right",
-                    "type": "string"
-                },
-                "paddingTop": {
-                    "title": "Padding top",
-                    "type": "string"
-                },
-                "paddingBottom": {
-                    "title": "Padding bottom",
-                    "type": "string"
-                },
-                "paddingLeft": {
-                    "title": "Padding left",
-                    "type": "string"
-                },
-                "paddingRight": {
-                    "title": "Padding right",
-                    "type": "string"
-                }
-            }
+        "color": {
```

Comment:
This is a breaking change, it should still have the data property with color property inside. From the linked task #2347:
> without changing its API